thavalon-webapp/src/components/Navbar.scss

```diff
@@ -0,0 +1,110 @@
+#navbarContainer {
+    background-color: #333;
+    overflow: hidden;
+
+    #rightContainer {
```

Comment:
What does nesting an id selector inside another id selector do in Sass?

Reply:
I _think_ it's the same as doing the following in CSS:

```css
#navbarContainer #rightContainer
```

Basically, rightContainer is a direct child of navbarContainer.

If it's prefixed with an &, then that means the element has both rightContainer and navbarContainer.